### DISCUSSION TITLE
message.loading 的loading动画颜色怎么调整与主题色相同

### DISCUSSION THREAD

**Discussion by chongdongkongjian** [ant-design/ant-design#44900] [Q&A] [answered]
<img width="677" alt="image" src="https://github.com/ant-design/ant-design/assets/127675292/47ce9527-75df-40c6-bae5-cf3e6ee5144e">
我应该怎么做

**Comment by Wxh16144** [comment, marked as answer]
colorInfo token 和 colorPrimary 是独立的，你可以用  colorInfo token 来修改
https://codesandbox.io/s/jia-zai-zhong-antd-5-9-1-forked-zpyfhn?file=/demo.tsx

<img width="1707" alt="image" src="https://github.com/ant-design/ant-design/assets/32004925/829e000a-a414-43d5-af72-3a607fe230aa">
